addons/mail/static/src/components/composer/composer_tests.js

```diff
@@ -165,6 +165,38 @@ QUnit.test('composer text input: basic rendering when linked thread is a mail.ch
     );
 });
 
+QUnit.test('extended composer: basic rendering', async function (assert) {
+    assert.expect(2);
+
+    Object.assign(this.data.initMessaging, {
+        channel_slots: {
+            channel_channel: [{
+                channel_type: 'channel',
+                id: 20,
+                is_pinned: true,
+                mass_mailing: true,
```

Comment:
~~Put comment that mailing channel have extended composer (already done in some other tests).~~

Actually, this is not a test of "extended" composer, but more of a mailing channel composer, which happens to use "extended" but I wouldn't use this concept when the test itself don't mention it.